md/create-or-modify-a-page.md

```diff
@@ -35,14 +35,19 @@ Custom widget is a special artifact that we will explain in a later chapter.
 
 ## Preview a page, layout, form, or fragment
 
-You can preview page, form and fragment rendering by clicking the **_Preview_** button. Another browser window opens with the artifact as it will be displayed after deployment. If you update the page, form or fragment, the preview is automatically refreshed when you save (not available for Internet Explorer 9).
+You can preview page, form and fragment rendering by clicking the **_Preview_** button. Another browser window opens with the artifact as it will be displayed after deployment. If you update the page, form or fragment, the preview is automatically refreshed when you save (not available for Internet Explorer 11).
+
+You can display your page, layout, form or fragment with one of your theme application installed. 
+
+You can open your preview in a browser tab with expand button, outside of current preview mechanism to allow you set new URL parameters, play with dynamic browser sizing, and to reuse same url in another browser to see how your form or page behaves in your user default browser.
```

Comment:
"You can open your preview in a browser tab with expand button, ...."
=> "From the preview window, you can also expand the preview outside of current preview mechanism, with the "Expand Preview in new window" button. This allows you to set new URL parameters ..."